src/webgl/texture.js

```diff
@@ -112,6 +112,13 @@ export const TEXTURE_FORMATS = {
   // [GL.COMPRESSED_RGBA_ATC_INTERPOLATED_ALPHA_WEBGL]: {compressed: true, gl1: ETC1}
 };
 
+const V3_TO_V4_OPTIONS_MAP = {
```

Comment:
>  I would opt for GL ENUM style 

Sounds good! Let's just add backwards compatibility support for the old opts then, with deprecation warnings.